### PR TITLE
Add updates from v0.4.2 in 2020/08 

### PR DESCRIPTION
**Pull request by mprevide**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -280,33 +176,53 @@ There is a dummy iotagent-http at samples/iotagent-base. There, you will find:
...
 
 ``` sh
 cd samples/iotagent-base
-sudo docker build -t dojot-training/iotagent-http .
+sudo docker build -t iotagent-http .
 cd -
 ```
 
-Then you need to regenerate the docker-compose.yml, to include this new microservice, and
+Then you need to add the text above to docker-compose.yml, to include this new microservice, and
```
Comment (thiagodpf) [inline README.md:191]: > Then you need to add the text **above** to docker-compose.yml

it wouldn't be: `below`? (This is if the yaml snippet is below the phrase when it is rendered)

**Comment by luissajunior** [comment]
My commits removed some informations from README. For example the cold chain instructions. That will go to master branch?

**Comment by mprevide** [comment]
> My commits removed some informations from README. For example the cold chain instructions. That will go to master branch?

Yes

**Comment by mprevide** [comment]
> My commits removed some informations from README. For example the cold chain instructions. That will go to master branch?

I was thinking about removing it, but now I'm thinking better and it might be useful.
Thx :)

File: README.md [hunk 2]
```diff
@@ -160,20 +185,66 @@ cd docker-compose
...
+TIP: If you want to stop dojot's microservices, run:
+
+
+Note: But if you stop dojot, you will lose all settings for template, device, flows and beyond history. In order not to ask for the template, device, flows settings, see more about importing and exporting at: https://dojotdocs.readthedocs.io/en/v0.4.2/using-web-interface.html#import-and-export
 
 ``` sh
 cd docker-compose
 sudo docker-compose down
 cd -
 ```
```
Comment (thiagodpf) [inline README.md]: The command `docker-compose down` could come right after the `tip` and then the `note` about the impacts of executing the command. In other words, the command might be between the tip and the note.

**Review by luissajunior — DISMISSED** [review]
That´s ok

**Pull request merged**